.github/workflows/firebasepod.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     # Don't run on private repo unless it is a PR.
     if: (github.repository == 'Firebase/firebase-ios-sdk' && github.event_name == 'schedule') || github.event_name == 'pull_request'
 
-    runs-on: macos-12
+    runs-on: macos-latest
```

Comment:
Could this be harder to debug (if something were to fail when `macos-latest` becomes macOS 13)? TIL I learned that:
> The `-latest` migration process is gradual and happens over 1-2 months in order to allow customers to adapt their workflows to the newest OS version. During this process, any workflow using the `-latest` label, may see changes in the OS version in their workflows or pipelines.

-- https://github.com/actions/runner-images#latest-migration-process

On the other hand, it does act as a canary for new macOS releases. I think either choice is reasonable.

Reply:
Yes, it was my intent to use this one as a canary.

Reply:
Perfect, thanks for clarifying!